From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: guineveresaenger <guinevere@pulumi.com>
Date: Wed, 22 Oct 2025 11:06:50 -0700
Subject: [PATCH] respelling of package


diff --git a/scm/shim.go b/scm/shim.go
index 0fabc68..4607cd8 100644
--- a/scm/shim.go
+++ b/scm/shim.go
@@ -1,8 +1,8 @@
 package scm
 
 import (
-	scm "github.com/PaloAltoNetworks/terraform-provider-scm/internal/provider"
 	"github.com/hashicorp/terraform-plugin-framework/provider"
+	scm "github.com/paloaltonetworks/terraform-provider-scm/internal/provider"
 )
 
 const version = "dev"
