From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: guineveresaenger <guinevere@pulumi.com>
Date: Wed, 22 Oct 2025 11:15:02 -0700
Subject: [PATCH] remove function call


diff --git a/scm/shim.go b/scm/shim.go
index 4607cd8..e573a39 100644
--- a/scm/shim.go
+++ b/scm/shim.go
@@ -8,5 +8,5 @@ import (
 const version = "dev"
 
 func Provider() provider.Provider {
-	return scm.New(version)()
+	return scm.New(version)
 }
